test/Riok.Mapperly.Tests/_snapshots/InstantiableMapperWithStaticMethodsTest.ShouldUseSimpleObjectFactory#CarMapper.g.verified.cs

```diff
@@ -0,0 +1,12 @@
+﻿//HintName: CarMapper.g.cs
+// <auto-generated />
+#nullable enable
+public partial class CarMapper
+{
+    static partial global::B Map(global::A a)
+    {
+        var target = new global::B();
```

Comment:
This doesn't look like what is expected, does it? The name of the test is `ShouldUseSimpleObjectFactory`. IMO this should be `var target = CreateB();`